### DISCUSSION TITLE
Add Perception DC to stealth tokens

### DISCUSSION THREAD

**Discussion by DaveParkinglot** [Saibot393/perceptive#83] [Ideas]
It would be so so so so helpful to be able to quickly see the Perception DC to see a given token.

My shitty mockup using paint:
![image](https://github.com/Saibot393/perceptive/assets/159180002/c0224ec7-cb73-438e-a3f8-98e58548b686)

**Comment by Saibot393** [comment]
Yes, that seems usefull, i will look into adding an option for that

**Comment by Saibot393** [comment]
I have added indicators for active and passive perception DCs with v3.5.0, you can select where to display them using the setting "Position of perception DC input", they can also be used to change the DC values. I can (currently) not display them in the middle though, since they would clash with the HP display
